src/main/java/com/renatusnetwork/momentum/data/stats/PlayerStats.java

```diff
@@ -44,6 +44,7 @@ public class PlayerStats
     private Clan clan;
     private Location currentCheckpoint;
     private Location practiceSpawn;
+    private Location practiceCP;
```

Comment:
I think we should rename the variables and getters to something more readable, maybe:
private Location practiceStart; (if practiceSpawn makes more sense to you, I am ok with that)
private Location practiceCheckpoint;

and then some setters/getters:
public void setPracticeCheckpoint(Location practiceCheckpoint) { this.practiceCheckpoint = practiceCheckpoint); }

public Location getPracticeCheckpoint() { return practiceCheckpoint; }

same for practiceStart (or practiceSpawn, up to you), you get the idea, just so it makes more sense without needing to look further

Reply:
this would mean going through the code for where getPracticeLocation is used and changing

Reply:
practiceCP -> practiceCheckpoint
setPracticeMode(Location loc) -> setPracticeMode(Location startLocation)
setPracticeCheckpoint(Location loc) -> setPracticeCheckpoint(Location checkpointLocation)